content/en/docs/failed-validation-limit.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Failed Validation Limit
+slug: failed-validation-limit
+top_graphic: 1
+lastmod: 2022-06-30
+show_lastmod: false
+---
+
+
+# Description
+All issuance requests are subject to a *Failed Validation* limit of 5 failures
+per account, per hostname, per hour. You should receive the following error
+message from your ACME client when you’ve exceeded the Failed Validation limit:
+
+```
+too many failed authorizations recently: see https://letsencrypt.org/docs/failed-validation-limit/
+```
+
+The ‘authorizations’ that this error refers to are the result of authorization
+requests, sent by your ACME client, to validate control over a domain name
+before we can issue or renew a certificate. This error indicates that the
+multiple requests for validation were sent successfully but all attempts to
+validate have failed.
+
+# Common Causes
+
+Subscribers who hit the Failed Validation limit often do so due to a
+misconfiguration in their environment.
+
+# HTTP-01 or TLS-APLN-01
```

Comment:
I'd make these second-level headings under "Common Causes"
```suggestion
## HTTP-01 or TLS-APLN-01
```

Reply:
It currently matches what we did for the previous rate limits page:
https://github.com/letsencrypt/website/blob/a8b414410bff3e016d3365d54131bb61b894f44c/content/en/docs/duplicate-certificate-limit.md?plain=1#L26

I believe I tried second-level headings on that one and they didn't look as good.

Reply:
Ah yeah, matching the other is reasonable. I think it looks good, though?
![image](https://user-images.githubusercontent.com/2474926/176788853-41686069-ebdc-48b0-bd68-05e59ac029bb.png)

Reply:
Oh, my bad! I initially read this as, "make all first-level headings second-level". Absolutely on-board, I'll make this change.